### ISSUE TITLE
Include button to screenshot current frame

### ISSUE THREAD

**Issue by TimonGaertner** [joschahenningsen/TUM-Live#1196]
It would be great to include a screenshot button, which copies the current frame of the presentation into your clipboard, so you can for example paste it into your notes.

**Comment by joschahenningsen** [comment]
Your OS already has this feature. Your browser probably as well. Any reason you can't use those?

**Comment by joschahenningsen** [comment]
Here's how to do it in firefox:


https://github.com/TUM-Dev/gocast/assets/44805696/b3d3bc67-63c2-429f-a2b1-95f134106e12

**Comment by YiranDuan721** [comment]
I also think it would be nice (but not necessary) to have this feature, which makes screenshoting more convenient (screenshot feature of the OS or the browser requires picking an area, which would take some time).

In addition, the screenshot feature integrated in TUM-live can include automatically adding a watermark on the picture, showing the name of the course, the name of the video and also the time position.

But would such a feature pose a copyright problem?

**Comment by TimonGaertner** [comment]
> Your OS already has this feature. Your browser probably as well. Any reason you can't use those?

Yes, I know, but that requires at least 4 clicks, an integrated button would decrease this action to 1 click. 
Additionally, if you use a tablet - Whether a 2-in-1 or a dedicated - The screenshot function of the os might not directly allow it to screenshot a specific part of the screen or it could be annoying. For example, if I use my surface in tablet mode it's really annoying to take a screenshot of only the presentation.
